src/plugins/intel_cpu/tests/unit/transformations/arm/deconv_1d_decomposition.cpp

```diff
@@ -0,0 +1,160 @@
+        if (p.with_output_shape) {
+            // Test 3-input variant with explicit output shape (only spatial dimensions)
+            ov::Shape output_shape;
+            if (rank == 3) {
+                // For 3D case, only specify the spatial dimension (L)
+                auto in_size = p.input_shape[2].is_static() ? p.input_shape[2].get_length() : 100;
+                auto kernel_size = p.weights_shape[2];
+                auto out_size = p.stride * (in_size - 1) + kernel_size;
+                output_shape = {static_cast<unsigned int>(out_size)};  // Only spatial dimension
+            } else {
+                output_shape = {100, 100};  // 4D case - two spatial dimensions
+            }
+            auto output_shape_const =
+                ov::opset1::Constant::create(ov::element::i64, {output_shape.size()}, output_shape);
+            deconv = std::make_shared<ov::opset1::ConvolutionBackpropData>(input,
+                                                                           weights,
+                                                                           output_shape_const,
+                                                                           strides,
+                                                                           pads,
+                                                                           pads,
+                                                                           dilations,
+                                                                           p.pad_type);
+        } else {
+            // Test 2-input variant
+            deconv = std::make_shared<ov::opset1::ConvolutionBackpropData>(input,
+                                                                           weights,
+                                                                           strides,
+                                                                           pads,
+                                                                           pads,
+                                                                           dilations,
+                                                                           p.pad_type);
+        }
+
+        model = std::make_shared<ov::Model>(ov::NodeVector{deconv}, ov::ParameterVector{input});
+
+        // Create reference model by applying the transformation on a clone
+        model_ref = model->clone();
+        if (p.expect_transformation) {
+            // Apply transformation to the reference model to get expected result
+            ov::pass::Manager ref_manager;
+            ref_manager.register_pass<ov::pass::InitNodeInfo>();
+            ref_manager.register_pass<Deconv1DDecomposition>();
+            ref_manager.run_passes(model_ref);
+        }
+        // If no transformation expected, model_ref stays as the clone (unchanged)
+
+        // Register the transformation to be tested
+        manager.register_pass<ov::pass::InitNodeInfo>();
```

Comment:
We don't need it ([discussion](https://github.com/openvinotoolkit/openvino/pull/31544#discussion_r2254140489))

Reply:
@v-Golubev fixed